docs/references/ic-interface-spec.md

```diff
@@ -590,23 +590,27 @@ The Internet Computer has two HTTPS APIs for canister calling:
 
 #### Asynchronous canister calling {#http-async-call-overview}
 
-1.  A user submits a call via the [HTTPS Interface](#http-interface). No useful information is returned in the immediate response (as such information cannot be trustworthy anyways).
+1.  A user submits a call via the [HTTPS Interface](#http-interface) and the call is received by a replica (a node belonging to an IC subnet). The receiving replica decides whether it accepts the call. An honest replica does so by checking that the target canister is not frozen and
 
-2.  For a certain amount of time, the IC behaves as if it does not know about the call.
+  - checking that the target canister is not empty and running and performing [ingress message inspection](#system-api-inspect-message) for calls to a regular canister;
 
-3.  The IC asks the targeted canister if it is willing to accept this message and be charged for the expense of processing it. This uses the [Ingress message inspection](#system-api-inspect-message) API for normal calls. For calls to the management canister, the rules in [The IC management canister](#ic-management-canister) apply.
+  - checking that the management canister method can be called via ingress messages and that the caller is a controller of the target canister for calls to the management canister.
```

Comment:
```suggestion
  - checking that the management canister method can be called via ingress messages and that the caller is a controller of the target canister for calls to the management canister.
  
  The replica's immediate response go the HTTPS call about the acceptance cannot be trusted, because a single replica may deviate from the protocol.
```

@mraszyk In the current version of the PR there is no information anymore about the response the sender gets immediately when submitting a call. I think it is important to mention something about this aspect. What do you think about my proposal to still write a sentence about this?

Reply:
That statement does not apply if the immediate response (when requesting /api/v3) contains a certificate for the response. There might also be no "immediate" response in case of /api/v3 endpoint. We could try to go more into details and list the cases which cannot be trusted, but I'm not sure if it's worthwhile.

Reply:
But this is in the part of the asynchronous call (v2), see to Line 591

Reply:
Sure, but the synchronous call part also refers to this section since the flow is similar (except that in many cases, the immediate response already contains the trustworthy certified response).

Reply:
How about: "Note that you can only trust valid IC certificates in IC responses since malicious replicas could spoof their responses."

Reply:
That's a great suggestion, thanks Martin!

Reply:
https://github.com/dfinity/portal/pull/5080/commits/d620aa8aca3f0daf9da59f5cde0ef9c7a50ec2f3